packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -444,6 +446,11 @@ export function authorizationServiceBuilder(
       );
 
       const client = await retrieveClient(clientId, readModelService);
+
+      if (client.data.kind !== ClientKind.Enum.Consumer) {
```

Comment:
```suggestion
      if (client.data.kind !== clientkind.consumer) {
```
You can use the already defined object, you could also extract this as an assert in `validators`